### DISCUSSION TITLE
simple triple sum possible?

### DISCUSSION THREAD

**Discussion by elistys** [wkpark/sd-webui-model-mixer#163] [Q&A]
The description of sd-webui-model-mixer says, "Doesn't support many of the calculation methods supported by SuperMerger."

I want to do the following simple triple sum that exists in SuperMerger, not the advanced merge. Is that possible? If so, please tell me where to set it.

Triple sum: A*(1-alpha-beta)+B*alpha+C*beta

I looked at the settings, but they were complicated and I couldn't understand them.

**Comment by wkpark** [comment]
Any series of A * alpha + B * beta + C * gamma .... could be translated into series of pairwise sum. for example
![image](https://github.com/user-attachments/assets/f80a7faa-f6b3-4cef-9997-c5b42c3501cd)

so,

$$ A * (1 - a - b) + B * a + C * b $$

could be translated into
- step-1 : A * (1 - x) + B * x - mix_1
- step-2 mix_1 * (1-y) *  C * y


$$ == (A * (1 - x) + B * x) * (1 - y) + C * y $$

and you can get x, y using `Weight Sum Calculator` (found at the `Model Block Weights` accordion)

**Reply by elistys** [reply]
Thank you for letting me know.

But I'm too stupid and could not understand your explanation and "Calculated alphas for given model weights for Weight Sum".

SuperMerger is easy because just set the ratio, so I can understand it.

I think most normal people can understand your explanation, so I'll close this thread here.

thank you.
